packages/stores/package.json

```diff
@@ -53,7 +53,7 @@
     "@cosmjs/math": "0.29.0",
     "@cosmjs/proto-signing": "0.29.0",
     "@cosmjs/stargate": "0.29.0",
-    "@cosmos-kit/core": "2.7.0",
```

Comment:
This version should match `web/package.json`

Reply:
oops great catch